packages/RichTextEditing-Core.package/DocumentReadStream.class/instance/on..st

```diff
@@ -0,0 +1,6 @@
+initialize
+on: aDocument
+
+	collection := aDocument text.
+	readLimit := collection size.
```

Comment:
Maybe set readLimit in the accessor for collection. Or is it necessary to set one but not the other?

Reply:
I think you should be able to change the text after opening a stream. So I would keep the way it is and also suggest to delete the setters for text and readLimit.